armi/operators/operator.py

```diff
@@ -622,6 +602,9 @@ def interactAllEveryNode(self, cycle, tn, excludedInterfaceNames=()):
 
     def interactAllEOC(self, cycle, excludedInterfaceNames=()):
         """Interact end of cycle for all enabled interfaces."""
+        if not math.isclose(self.r.p.availabilityFactor, 1):
+            self.r.p.time += self.r.p.cycleLength * (1 - self.r.p.availabilityFactor) / units.DAYS_PER_YEAR
```

Comment:
I do not think there is any benefit to wrapping this in an `if` statement. Just update `time` every time, no matter what `availabilityFactor` is.

Reply:
I just copied this over. But that's an interesting point.